Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/zsh
+# This script is used to publish the 2 function apps into 2 zip files, for later azure deployment.
+# no input needed, and it could be run from any where.
+
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+dotnet clean
+dotnet publish -c Release
+PUBLISH_DIR="./bin/Release/net6.0/publish/"
+
+find "$SCRIPTPATH"/ -type f -iname \*.csproj | while read ss; \
+do \
+    SCRIPT=$(realpath "$ss")
+    CSPROJDIR=$(dirname "$SCRIPT")
+    cd "$CSPROJDIR"
+
+    filename=$(basename -- "$ss")
+    extension="${filename##*.}"
+    filename="${filename%.*}"
+    cd "$PUBLISH_DIR"
+    zip -r ./"$filename".zip ./*
+    mv -fv ./"$filename".zip "$CSPROJDIR/../Package/"
+    cd ../
+done
```

Comment:
Clean the bin folder at the end of this script? We should cleanup any temporary directories we use at the end of the script.

Reply:
the dotnet clean will clean it.
no need to clean every time, as this will increase the build/compilation time..

Reply:
The bin directory is still present. It looks like since all the files in the bin directory are part of gitignore, we don't see it. I don't mind it then.